causing/indiv.py

```diff
@@ -4,59 +4,33 @@
 from numpy import median, zeros
 
 
-def compute_indiv_row(i, xy_dim, model_dat):
-    """compute single row of dx_mat or dy_mat, subtracting median observation,
-    if x_basevars, y_basevars are given, data are scaled by their base_var"""
```

Comment:
So where does the division by basevar takes place now?

Reply:
In a preprocessing step. We decided that together when removing rat_var and rel_var. See https://github.com/realrate/RealRate-Private/pull/138